test/core/end2end/tests/retry_streaming.cc

```diff
@@ -59,345 +45,131 @@ static std::unique_ptr<CoreTestFixture> begin_test(
-  auto f = begin_test(config, "retry_streaming", &client_args, nullptr);
-
-  grpc_core::CqVerifier cqv(f->cq());
-
-  gpr_timespec deadline = grpc_timeout_seconds_to_deadline(5);
-  c = grpc_channel_create_call(f->client(), nullptr, GRPC_PROPAGATE_DEFAULTS,
-                               f->cq(),
-                               grpc_slice_from_static_string("/service/method"),
-                               nullptr, deadline, nullptr);
-  grpc_core::channelz::ChannelNode* channelz_channel =
-      grpc_channel_get_channelz_node(f->client());
-
-  GPR_ASSERT(c);
-
-  peer = grpc_call_get_peer(c);
-  GPR_ASSERT(peer != nullptr);
-  gpr_log(GPR_DEBUG, "client_peer_before_call=%s", peer);
-  gpr_free(peer);
-
-  grpc_metadata_array_init(&initial_metadata_recv);
-  grpc_metadata_array_init(&trailing_metadata_recv);
-  grpc_metadata_array_init(&request_metadata_recv);
-  grpc_call_details_init(&call_details);
-  grpc_slice status_details = grpc_slice_from_static_string("xyz");
-
+TEST_P(RetryTest, RetryStreaming) {
+  InitServer(ChannelArgs());
+  InitClient(
+      ChannelArgs()
+          .Set(GRPC_ARG_MAX_CHANNEL_TRACE_EVENT_MEMORY_PER_NODE, 1024 * 8)
+          .Set(GRPC_ARG_ENABLE_CHANNELZ, true)
+          .Set(GRPC_ARG_SERVICE_CONFIG,
+               "{\n"
+               "  \"methodConfig\": [ {\n"
+               "    \"name\": [\n"
+               "      { \"service\": \"service\", \"method\": \"method\" }\n"
+               "    ],\n"
+               "    \"retryPolicy\": {\n"
+               "      \"maxAttempts\": 3,\n"
+               "      \"initialBackoff\": \"1s\",\n"
+               "      \"maxBackoff\": \"120s\",\n"
+               "      \"backoffMultiplier\": 1.6,\n"
+               "      \"retryableStatusCodes\": [ \"ABORTED\" ]\n"
+               "    }\n"
+               "  } ]\n"
+               "}"));
+  auto c =
+      NewClientCall("/service/method").Timeout(Duration::Seconds(5)).Create();
+  channelz::ChannelNode* channelz_channel =
+      grpc_channel_get_channelz_node(client());
+  EXPECT_NE(c.GetPeer(), absl::nullopt);
   // Client starts a batch for receiving initial metadata, a message,
   // and trailing metadata.
-  memset(ops, 0, sizeof(ops));
-  op = ops;
-  op->op = GRPC_OP_RECV_INITIAL_METADATA;
-  op->data.recv_initial_metadata.recv_initial_metadata = &initial_metadata_recv;
-  op++;
-  op->op = GRPC_OP_RECV_MESSAGE;
-  op->data.recv_message.recv_message = &response_payload_recv;
-  op++;
-  op->op = GRPC_OP_RECV_STATUS_ON_CLIENT;
-  op->data.recv_status_on_client.trailing_metadata = &trailing_metadata_recv;
-  op->data.recv_status_on_client.status = &status;
-  op->data.recv_status_on_client.status_details = &details;
-  op++;
-  error = grpc_call_start_batch(c, ops, static_cast<size_t>(op - ops),
-                                grpc_core::CqVerifier::tag(1), nullptr);
-  GPR_ASSERT(GRPC_CALL_OK == error);
-
+  CoreEnd2endTest::IncomingStatusOnClient server_status;
```

Comment:
Here's another file with the unnecessary `CoreEnd2endTest::` prefix.  Might be a good idea to just scan all of the files and eliminate this prefix wherever it exists.

Reply:
Maybe. Do note that it causes no harm too.